tm2/pkg/sdk/baseapp.go

```diff
@@ -409,12 +410,20 @@ func handleQueryApp(app *BaseApp, path []string, req abci.RequestQuery) (res abc
 			} else {
 				result = app.Simulate(txBytes, tx)
 			}
+
 			res.Height = req.Height
-			res.Value = amino.MustMarshal(result)
+
+			bytes, err := json.Marshal(result)
```

Comment:
Why not `amino JSON`?

Reply:
Oh I changed it, I now use amino.MarshalJSON.